File: 108-1 独特的电子邮件地址/64ms.py
class Solution:
    def numUniqueEmails(self, emails):
        """
        :type emails: List[str]
        :rtype: int
        """
        addr=set()
        for s in emails:
            name,domain=s.split('@')
            name=name.replace('.','').split('+')[0]
            addr.add(name+'@'+domain)
        return len(addr)

File: 116-1 重复 N 次的元素/64ms.cpp
class Solution {
public:
    int repeatedNTimes(vector<int>& A) {
        int l=A.size();
        int n=l/2;
        map<int,int> m;
        for(auto i:A){
            m[i]++;
            if(m[i]==n) return i;
        }
        return 0;
    }
};
